README.md

```diff
@@ -79,6 +84,39 @@ export class MyAppModule {}
 - `hydratedStateKey`: if present, the sync reducer will add a flag to your app state when the initial hydration has completed. You can bind an observable to this flag, and so be notified when the app has been hydrated.
 - `onSyncError`: a callback, called whenever there was an error syncing the state. The callback is passed a single `err` parameter.
 
+## If main reducer is an ActionReducerMap
+
+`hydrated.reducer.ts`
+```js
+import { Action } from '@ngrx/store';
+
+export interface State {}
```

Comment:
Personal preference, but could you rename this to `HydratedState` so it doesn't have the same name as the state in the main reducer.